diff --git a/vendor/symfony/console/Input/InputInterface.php b/vendor/symfony/console/Input/InputInterface.php
index 8efc6232..0b6c1b36 100644
--- a/vendor/symfony/console/Input/InputInterface.php
+++ b/vendor/symfony/console/Input/InputInterface.php
@@ -26,7 +26,7 @@ interface InputInterface
      *
      * @return string|null The value of the first argument or null otherwise
      */
-    public function getFirstArgument();
+    public function getFirstArgument(): ?string;
 
     /**
      * Returns true if the raw parameters (not parsed) contain a value.
@@ -41,7 +41,7 @@ public function getFirstArgument();
      *
      * @return bool true if the value is contained in the raw parameters
      */
-    public function hasParameterOption($values, $onlyParams = false);
+    public function hasParameterOption(string|array $values, bool $onlyParams = false): bool;
 
     /**
      * Returns the value of a raw option (not parsed).
@@ -57,7 +57,7 @@ public function hasParameterOption($values, $onlyParams = false);
      *
      * @return mixed The option value
      */
-    public function getParameterOption($values, $default = false, $onlyParams = false);
+    public function getParameterOption(string|array $values, string|bool|int|float|array|null $default = false, bool $onlyParams = false);
 
     /**
      * Binds the current Input instance with the given arguments and options.
@@ -78,7 +78,8 @@ public function validate();
      *
      * @return array<string|bool|int|float|array|null>
      */
-    public function getArguments();
+    public function getArguments(): array;
+
 
     /**
      * Returns the argument value for a given argument name.
@@ -108,14 +109,16 @@ public function setArgument($name, $value);
      *
      * @return bool true if the InputArgument object exists, false otherwise
      */
-    public function hasArgument($name);
+    public function hasArgument($name): bool;
+
 
     /**
      * Returns all the given options merged with the default values.
      *
      * @return array<string|bool|int|float|array|null>
      */
-    public function getOptions();
+    public function getOptions(): array;
+
 
     /**
      * Returns the option value for a given option name.
@@ -145,14 +148,16 @@ public function setOption($name, $value);
      *
      * @return bool true if the InputOption object exists, false otherwise
      */
-    public function hasOption($name);
+    public function hasOption($name): bool;
+
 
     /**
      * Is this input means interactive?
      *
      * @return bool
      */
-    public function isInteractive();
+    public function isInteractive(): bool;
+
 
     /**
      * Sets the input interactivity.
